### ISSUE TITLE
Unable to use react-flow with Vite or Astro

### ISSUE THREAD

**Issue by lloydjatkinson** [wbkd/react-flow#3384]
### Describe the Bug

```
[vite] Error when evaluating SSR module /src/components/Flow.tsx: failed to import "reactflow"
|- /home/projects/github-2jb4w6-ws5mu8/node_modules/reactflow/dist/esm/index.js:1
export * from '@reactflow/core';
^^^^^^

SyntaxError: Unexpected token 'export'
    at Object.compileFunction (https://github2jb4w6ws5mu8-xuv3.w-credentialless.staticblitz.com/blitz.7259293c.js:72:4777)
    at wrapSafe (https://github2jb4w6ws5mu8-xuv3.w-credentialless.staticblitz.com/blitz.7259293c.js:56:13586)
    at Module._compile (https://github2jb4w6ws5mu8-xuv3.w-credentialless.staticblitz.com/blitz.7259293c.js:56:13951)
    at Module._extensions..js (https://github2jb4w6ws5mu8-xuv3.w-credentialless.staticblitz.com/blitz.7259293c.js:56:14989)
    at Module.load (https://github2jb4w6ws5mu8-xuv3.w-credentialless.staticblitz.com/blitz.7259293c.js:56:13005)
    at Module._load (https://github2jb4w6ws5mu8-xuv3.w-credentialless.staticblitz.com/blitz.7259293c.js:56:9949)
```

### Your Example Website or App

https://stackblitz.com/edit/github-2jb4w6-ws5mu8?file=astro.config.mjs,package.json,src%2Fpages%2Findex.astro&on=stackblitz

### Steps to Reproduce the Bug or Issue

1. Copy paste example from getting started pages in docs
2. Use it in a React component
3. Import react component
4. Observe how ES modules are not being imported correctly

### Expected behavior

As a user I expect to be able to use imports from react-flow in my React components when using Astro (built on top of Vite).

### Platform

Chrome. Node 20.x

**Comment by DonMiller9294** [comment]
Here try these trouble shooting options.



To get past this problem, let's focus on fixing the issue with how you're bringing in the "reactflow" module into your "/src/components/Flow.tsx" file. Here's what you can do in simpler terms:

1. **Check Your Package**: Make sure you've got the right version of the "reactflow" package installed in your project. Think of it like making sure you have the right tool for the job, and that it's listed in your project's "package.json" file.

2. **Read the Manual**: Take a look at the documentation or any notes that came with the "reactflow" package. Sometimes, there are specific rules or settings you need to follow when you're doing server-side rendering.

3. **Compatibility Check**: Make sure your project is speaking the same language as "reactflow." If "reactflow" is using a newer kind of code (ES6 modules), make sure your project understands it. You might need to tweak your project settings to make things match.

4. **Importing the Right Way**: Review how you're importing "reactflow" in your "/src/components/Flow.tsx" file. It's like making sure you're opening the right door with the right key. Confirm that your import statement is spot-on and follows the right format.

**Comment by lloydjatkinson** [comment]
@DonMiller9294 I can't believe you've got a ChatGPT bot going round uselessly commenting garbage on real github issues.

**Comment by bcakmakoglu** [comment]
Can you provide a reproduction for this issue?
[Seeing as the examples in the repo](https://github.com/wbkd/react-flow/tree/main/examples/vite-app) (not the docs) are built with vite, i don't really see how vite would be the issue here.

**Comment by DonMiller9294** [comment]
@lloydjatkinson First off it's not a chat gpt bot. I did some research on the internet this is what I found on a forum. I don't know why your being rude, all I am trying to do is help out.

**Comment by moklick** [comment]
This should be fixed in 11.9.0 - working example: https://github.com/wbkd/react-flow-example-apps/tree/main/reactflow-astro
